src/test/ui/closures/2229_closure_analysis/run_pass/multilevel-path-1.rs

```diff
@@ -0,0 +1,29 @@
+// run-pass
```

Comment:
it'd be useful to have a comment indicating what is being tested here 

Seems like the key difference is that it *writes* `w.p.x`